### DISCUSSION TITLE
Showing nodejs icon when it shouldnt

### DISCUSSION THREAD

**Discussion by micah** [starship/starship#5073] [Q&A] [answered]
I just installed starship, and when I'm in my home directory, it shows the nodejs icon, and if I do `startship explain` it says:
```
 "via  " (<1ms)        -  The currently installed version of NodeJS
```

I don't have nodejs installed, so I'm wondering why it says that. I dont particularly want a nodejs icon to be displayed when I'm in my home directory, so would love to be able to make that go away!

**Comment by davidkna** [comment, marked as answer]
By default, the Node.js module display when [any of the documented conditions](https://starship.rs/config/#node-js) are met (most commonly `.js` files). You can try disabling the module or restricting the `detect_` options. To avoid displaying modules if the runtime isn't installed, please use the [no empty icons preset](https://starship.rs/presets/no-empty-icons.html).
